docs/user/security/api-keys/index.asciidoc

```diff
@@ -23,51 +22,47 @@ image:images/api-keys.png["API Keys UI"]
 [[api-keys-security-privileges]]
 === Security privileges
 
-You must have the `manage_security`, `manage_api_key`, or the `manage_own_api_key` 
-cluster privileges to use API keys in {kib}.  API keys can also be seen in a readonly view with access to the page and the `read_security` cluster privilege. To manage roles, open the main menu, then click 
-*Stack Management > Roles*, or use the <<role-management-api, {kib} Role Management API>>. 
+You must have the `manage_security`, `manage_api_key`, or the `manage_own_api_key` cluster privileges to use API keys in {kib}.  
+
+*Personal API key*: In order to create or update a personal API key users need the `manage_security` privilege.
```

Comment:
Only `manage_api_key` or `manage_own_api_key` privileges are required to manage Personal API keys